client/python/cli/polaris_cli.py

```diff
@@ -118,7 +118,7 @@ def _get_client_builder(options):
         # Validates
         has_access_token = options.access_token is not None
         has_client_secret = client_id is not None and client_secret is not None
-        if has_access_token and has_client_secret:
+        if has_access_token and (options.client_id or options.client_secret):
```

Comment:
Not a blocker: i'm OK even both are configured. We could let it takes token first, if not found, then try to find client id and secrets. Only failed when neither is available.